hadoop-hdds/docs/content/interface/S3.zh.md

```diff
@@ -111,6 +113,23 @@ export AWS_SECRET_ACCESS_KEY=c261b6ecabf7d37d5f9ded654b1c724adac9bd9f13e247a235e
 aws s3api --endpoint http://localhost:9878 create-bucket --bucket bucket1
 ```
 
+## 公开任何卷
+
+与 S3 相比，Ozone 在名称空间层次结构中还有一个元素：卷。 默认情况下，可以使用 S3 接口访问 `/s3v` 卷的所有存储桶，但仅显示 `/s3v` 卷的（Ozone）存储桶。
+
+为了使 S3 接口上的任何其他桶可用，可以创建一个“符号链接”的桶:
+
+```bash
+ozone sh create volume /s3v
+ozone sh create volume /vol1
+
+ozone sh create bucket /vol1/bucket1
```

Comment:
```suggestion
ozone sh volume create /s3v
ozone sh volume create /vol1

ozone sh bucket create /vol1/bucket1
```

Reply:
This has been addressed in HDDS-4662 (#1769) , so it needs to be fixed here too.

Reply:
Solve it. :smile: